content/monitors/monitor_uptime_widget.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Monitor Uptime Widget
+kind: documentation
+description: "Track your monitor’s uptime via a widget in your Screenboard"
+private: true
+disable_toc: true
+---
+
+<div class="alert alert-warning">
+The Monitor Uptime Widget is currently in private beta. Interested in trying it out? <a href="https://goo.gl/forms/KmWpQebCeeOdpPAq1">Request private beta access</a>.
+</div>
+
+{{< img src="monitors/monitor_uptime_widget.png" alt="monitor uptime widget" responsive="true" >}}
+
+Use the monitor Uptime widget to track your monitor’s uptime via a widget in your [Screenboards][1]. Monitor Uptime is defined as the amount of time a monitor was in an up state (OK) compared to down state (non-OK). Similar to the monitor status page visualization, the status is represented in bars as green (up) and red (down).
```

Comment:
"Monitor Uptime Widget" is camel-cased on [line 10](https://github.com/DataDog/documentation/pull/3235/files#diff-b36fb08e1ce89cb3a44564646f416a5fR10) but not here. I don't have strong opinions on capitalisation in general, but we should be consistent within the same document at least. 😄

Reply:
Uptime should be capitalised since it's the name of the widget, the rest should be all lower case